src/app/shared/team-radio-switch/team-radio-switch.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="team-body">
+  <label for="team1" [ngClass]="refInput2.checked ? 'unchecked' : ''">
```

Comment:
To avoid using of an empty else value you may use `[ngClass]="{'unchecked': refInput2.checked}"`

Reply:
done, thanks